tasks.php

```diff
@@ -0,0 +1,23 @@
+<?php
+header('Content-Type: application/json; charset=utf-8');
+header("Access-Control-Allow-Origin: *");
+require 'vendor/autoload.php';
+
+try {
+    $db = \ProjectManager\Services\DbConnector::connect();
+    $data = \ProjectManager\Hydrators\TasksHydrator::getTasksByUserAndProjectId($db);
+        if (is_int(['project_id']) && ['project_id'] > 0 && is_int(['user_id']) && ['user_id'] > 0) {
```

Comment:
we used is_numeric in story  as it seemed to fix an issue we were having. not saying it is right but just letting you know